CHANGELOG.md

```diff
@@ -72,7 +72,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
 * [FEATURE] Distributor, ingester: ingestion of native histograms. The new per-tenant limit `-ingester.native-histograms-ingestion-enabled` controls whether native histograms are stored or ignored. #4159
 * [FEATURE] Query-frontend: Introduce experimental `-query-frontend.query-sharding-target-series-per-shard` to allow query sharding to take into account cardinality of similar requests executed previously. This feature uses the same cache that's used for results caching. #4121 #4177 #4188 #4254
 * [ENHANCEMENT] Go: update go to 1.20.1. #4266
-* [ENHANCEMENT] Ingester: added `out_of_order_blocks_external_label_enabled` shipper option to label out-of-order blocks before shipping them to cloud storage. #4182 #4297
+* [ENHANCEMENT] Ingester: added `out_of_order_blocks_external_label_enabled` shipper option to label out-of-order blocks before shipping them to cloud storage. #4182 #4297 #4440
```

Comment:
I'm guessing this will make it to `2.7.0`, but I don't know enough about the release process to know if editing the changelog here (`2.7.0-rc.0`) is the right thing to do.

Reply:
We should not extent changelog entries from existing releases. This change needs changelog entry in `main / unreleased` section now.

Reply:
Done here [`db9c497` (#4440)](https://github.com/grafana/mimir/pull/4440/commits/db9c4979100c2a0fde3d869d668e5357f5c0aeef)

Just curious: if one wanted to include a hotfix in `2.7.0`, what's the right way to update the changelog? Add a new secion `2.7.0-rc.1` and then all the `2.7.0-rc.*` entries get combined into `2.7.0` once the release is done?

Reply:
It's an experimental feature, so no need to cherry pick it in 2.7. Thanks!

Reply:
> Add a new secion 2.7.0-rc.1 and then all the 2.7.0-rc.* entries get combined into 2.7.0 once the release is done?

Yes, once final release is done, we collapse all individual -rc sections.